### PR TITLE
Add Whisper example

### PR DESCRIPTION
**Pull request by attila-dusnoki-htec**
This example show how to use HuggingFace's [Whisper](https://huggingface.co/openai/whisper-tiny.en) model with migraphx's python frontend.

A screenshot of the Gradio vesrion:
gradio_asr

### CODE REVIEW

File: examples/transformers/python_whisper/whisper.ipynb [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+    "    return processor(sample_data,\n",
+    "                     sampling_rate=sampling_rate,\n",
+    "                     return_tensors=\"np\").input_features"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will encode these, and use them in the decoding step."
```
Comment (kahmed10) [inline examples/transformers/python_whisper/whisper.ipynb:217]:
```suggestion
    "We will encode these and use them in the decoding step."
```

**Comment by kahmed10** [comment]
@attila-dusnoki-htec very minor grammatical change, but otherwise LGTM from code side.

@causten to verify example runs

File: examples/transformers/python_whisper/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#####################################################################################
+accelerate
+datasets
+optimum[onnxruntime]
+pydub
```
Comment (kahmed10) [inline examples/transformers/python_whisper/requirements.txt:27]: missing `soundfile`

File: examples/transformers/python_whisper/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#####################################################################################
+accelerate
+datasets
```
Comment (kahmed10) [inline examples/transformers/python_whisper/requirements.txt:25]: missing `librosa`

**Comment by gyulaz-htec** [comment]
@kahmed10 added the missing dependencies you mentioned

**Review by kahmed10 — APPROVED** [review]

**Review by causten — APPROVED** [review]

**Pull request merged**